🎉 Created payment_status.prisma

Artorisity opened the pull request:
## 📋 Summary

Created a Payment Status model.

## ✨ Description

This is in line with the construction of the tables and/or models that are related to the Payment module, following the concern on verifying of the non-nullability of the "created_at" and "updated_at" attributes as identified in the Data Dictionaries within the STP document.

## 🔧 Changes

1. Created "payment_status.prisma."

## 🔗 Related Issue

- Closes #38

## ✅ Checklist

- [ ] 🧹 Code follows the project's coding standards.
- [ ] 📚 Documentation has been updated (if necessary).
- [ ] 🧪 Tests have been written or updated (if applicable).
- [ ] ✅ All tests pass successfully.

## 💬 Additional Notes

Mostly the same concerns with the payment method model [#36] (https://github.com/ICS26010-group-3/twin-cj/issues/36)

## Notes for Reviewers

N/A.

Artorisity commented:
The comments in the source code (// In the STP document, the Data Dictionary signifies that it [1] accepts null values [2] it is not required.) is actually wrong - instead of the STP document, it's actually in the SDD document.

wjamesdevera commented:
already created the transaction model please add its relationship with the payment status

Artorisity commented:
@wjamesdevera initiated new commits and pushes to the sub-branch.

**Changes**
- Changed file name of "transaction_model.prisma" to "transaction.prisma."
- Added a one-to-many relationship between payment_status and transaction.
